css/style.css

```diff
@@ -0,0 +1,342 @@
+/*
+Project Name: Transition to Employment Toolkit
+Client: Transition to Employment
+Author: Hyun Masiello 
+*/
+
+
+/******************************************
+/* SETUP
+/*******************************************/
+
+/* Box Model Hack */
+* {
+     -moz-box-sizing: border-box; /* Firexfox */
+     -webkit-box-sizing: border-box; /* Safari/Chrome/iOS/Android */
+     box-sizing: border-box; /* IE */
+}
+
+.clearfix:before,
+.clearfix:after {
+  content: "";
+  display: table;
+}
+
+.clearfix:after {
+  clear: both;
+}
+
+.clearfix {
+  zoom: 1; /* ie 6/7 */
+}
+
+/******************************************
+/* BASE STYLES
+/*******************************************/
+/*
+colors: 
+
+*/
+
+body {
+  padding: 0px 15px;
+}
+
+header {
+  padding-top: 50px;
+}
+
+a {
+  color:#245E87;
+  text-decoration: none;
+}
+
+a:hover {
+  color: #08263A ;
+}
+
+nav {
+  font-family: Arial;
+  font-size: 1.5em;
+  padding: 115px 0px 70px 0px;
+}
+
+nav ul {
+  list-style-type: none;
+  position: relative;
+}
+
+nav ul ul {
+  display: none;
+}
+
+nav ul li {
+  padding: 15px 0px 0px 30px;
+}
+
+nav ul ul li {
+  background: white;
+  display: block;
+  padding: 15px 10px;
+}
+
+img {
+  max-width: 100%;
+}
+
+h1 {
+  color: #245E87;
+  font-family: Arial Narrow;
+  font-weight: 800;
+  font-size: 1.8em;
+  line-height: 1.5;
+  padding: 40px 0px 10px 0px;
+}
+
+h2 {
+  color: #245E87;
+  font-family: Arial;
```

Comment:
If you put `Arial` as your default font up in the `html` selector (see above), you can remove all these `font-family` values since your header tags will just inherit the `font-family` from `html`